### PR TITLE
Hide Indicator PR

### PR DESCRIPTION
**Pull request by adi90x**
Hello, 

As discussed in #114  , a quick PR to be able to hide an indicator based on a function result.
==> Give a way to display indicator only in certains circonstances.

Regards,
Adrien

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,6 +143,7 @@ Please ⭐️ this repo if you find it useful
 | indicators: `name:source:attribute` | string | optional | v1.0.1 | entity attribute
 | indicators: `name:source:mapper` | function | optional | v1.0.1 | value processing function
 | indicators: `name:tap_action` | [action object](#tap-action-object) | true | v1.1.0 | Action on click/tap.
+| indicators: `name:disabled` | function | optional | v2.4.5 | disabled/enabled rendering of indicators
```
Comment (regevbr) [inline README.md:146]: let's use `hide` instead and change the description accordingly

File: src/main.js [hunk 2]
```diff
@@ -308,6 +308,9 @@ class MiniClimate extends LitElement {
         item.functions.icon.style = compileTemplate(item.icon.style, context);
     }
 
+    if (item.disabled)
```
Comment (regevbr) [inline src/main.js:311]: lets add support for boolean as well, and extend this ability to other hide functionality in other componenets

Comment (adi90x) [reply src/main.js:311]: How do you handle those ?

File: README.md (hunk 1 above)
Comment (regevbr) [reply README.md:146]: I don't see it used anywhere in the indicators context, and if it already used, lets expend it to be either boolean or function

File: src/main.js (hunk 2 above)
Comment (regevbr) [reply src/main.js:311]: You can check the type of the value given, if it is boolean, just return a function that returns that value

**Comment by regevbr** [comment]
@adi90x if you want, I can take this PR and complete it myself as I want to take this chance to make this feature bigger and apply it to all components.

**Comment by adi90x** [comment]
Yes no problem it is all yours 😀

**Comment by regevbr** [comment]
done in #118 and released in 2.5.0.
@adi90x would appreciate if you can verify that it works well  - specifically that it reacts dynamically

**Pull request closed without merging**